utils/fetchBridgeDepositStatus.ts

```diff
@@ -0,0 +1,35 @@
+import { BridgeStatus } from "@/types";
+import { getERC20PegContract } from "@/utils";
+import { Api } from "@cennznet/api";
+import { ethers } from "ethers";
+
+// TODO: Needs test
+export default async function fetchBridgeDepositStatus(
+	api: Api,
+	provider: ethers.providers.Web3Provider
+): Promise<BridgeStatus> {
+	const pegContract = getERC20PegContract<"ReadOnly">(provider);
+
+	const [bridgePaused, cennzActive, ethActive] = await Promise.all([
+		api.query.ethBridge.bridgePaused(),
```

Comment:
we don't need to check if bridgePaused from cennznet side..
https://github.com/cennznet/bridge-contracts/pull/70

Reply:
addressed in https://github.com/cennznet/app-hub/pull/139